### PR TITLE
De novo transcriptome assembly, annotation, and differential expression analysis

### LINKED ISSUE #59: RNASeq: de novo transcriptome assembly with Trinity
For the #58 hackathon, I propose to work on some training material for de novo transcriptome assembly with Trinity.
The Trinity wrappers are up to date in IUC, and I have a (not yet public) workflow using them to do assembly+abundance estimation+differential expression.
Any help is welcome of course!

### PR DESCRIPTION
**Pull request by abretaud**
Finally a tutorial for de novo transcriptome assembly, annotation, and differential expression analysis as promised in #59 .

It's different than the existing [de novo transcriptomics](https://training.galaxyproject.org/training-material/topics/transcriptomics/tutorials/de-novo/tutorial.html) as here we don't use a reference genome, we just assemble the transcriptome using Trinity

We (with @r1corre) have a first draft of workflow, but a few things are missing:
- blast & co process before trinotate
- trinotate visualization
- better handling of data collections probably
- find a good dataset
- ...

Fixes #59

### CODE REVIEW

**Comment by lecorguille** [comment]
It's still in progress.

TODO : 
- [ ] Finish DE part
- [ ] Annotation part
- [ ] Text, Text, Question ...
- [ ] Main `questions`
- [ ] Main `objectives`
- [ ] Main `key_points`
- [ ] Satellite
  - [ ] `data-library.yaml`
  - [ ] `tutorial.bib`
- [ ] ... 
- [ ] Full reading

**Comment by lecorguille** [comment]
Can we consider to merge this PR? 
I disabled the tutorial to get a nice URL we will be able to share tomorrow. 

I promise to daily beat @r1corre to make him finish this tutorial 👿

File: CONTRIBUTORS.yaml [hunk 1]
```diff
@@ -540,6 +546,10 @@ workflow4metabolomics:
 wm75:
     name: Wolfgang Maier
 
+xiliu:
+    name: Xi Liu
+    github: false
```
Comment (bgruening) [inline CONTRIBUTORS.yaml]: can you remove this @lecorguille

Comment (lecorguille) [reply CONTRIBUTORS.yaml]: AH ... how can I add Xi as author?

**Comment by shiltemann** [comment]
thanks @abretaud and @lecorguille! and good luck tomorrow!

**Comment by lecorguille** [comment]
Thanks!

**Comment by vasudhajain29** [comment]
hi, I am trying to run denovo transcriptome pipeline on usegalaxy.org, but the issue is I cant see Trinity in the assembly method. Can you please guide me through how to use it?

Thanks

**Pull request merged**